pkg/controller/federatedtypeconfig/controller.go

```diff
@@ -104,7 +105,7 @@ func ProvideController(arguments args.InjectArgs, fedNamespace, clusterNamespace
 }
 
 func (c *FederatedTypeConfigController) Reconcile(k types.ReconcileKey) error {
-	log.Printf("Running reconcile FederatedTypeConfig for %s\n", k.Name)
+	glog.Infof("Reconciling FederatedTypeConfig %v/%v", k.Namespace, k.Name)
```

Comment:
(no action required) We can just print out `k` here with one format specifier as it has a `String` method that prints it as `namespace/name` if it contains a namespace. See the example on line 113 below.